packages/cli/src/commands/generate-block-types.ts

```diff
@@ -116,9 +116,9 @@ const generateBlockTypes = new Command("generate-block-types")
                 content += `export interface ${block.name}BlockInput {\n`;
                 block.inputFields.forEach((field) => {
                     content += field.name;
-                    if (field.nullable) content += "?";
                     content += ": ";
                     writeFieldType(field, "BlockInput");
+                    if (field.nullable) content += " | null";
```

Comment:
For inputs GraphQL is `argument?: <type> | null`, i.e. it supports both `undefined` and `null`. Should we do it the same way?

Reply:
I would say no